### PR TITLE
[15.0][ADD] fieldservice_person_salary_skill

### PR DESCRIPTION
**Pull request by LucPinheiro**
@dalonsod 
@ChristianSantamaria

### CODE REVIEW

**Review by dalonsod — APPROVED** [review]
Code review and tested 👍

File: fieldservice_person_salary_skill/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-solvosci%2Fslv--field--service-lightgray.png?logo=github
+    :target: https://github.com/solvosci/slv-field-service/tree/15.0/fieldservice_person_salary_skill
+    :alt: solvosci/slv-field-service
+
+|badge1| |badge2| |badge3| 
+
+ Adds skills to Person Salary for FSM orden.
```
Comment (ChristianSantamaria) [inline fieldservice_person_salary_skill/README.rst]:
```suggestion
 Adds skills to Person Salary for FSM order.
```

File: fieldservice_person_salary_skill/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "FieldService Person Salary Skill",
+    "summary": """
+        Adds skills to Person Salary for FSM ordens
```
Comment (ChristianSantamaria) [inline fieldservice_person_salary_skill/__manifest__.py]:
```suggestion
        Adds skills to Person Salary for FSM orders
```

File: fieldservice_person_salary_skill/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+ Adds skills to Person Salary for FSM orden.
```
Comment (ChristianSantamaria) [inline fieldservice_person_salary_skill/readme/DESCRIPTION.rst]:
```suggestion
 Adds skills to Person Salary for FSM order.
```

File: fieldservice_person_salary_skill/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    """,
+    "author": "Solvos",
+    "license": "AGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Field Service",
+    "website": "https://github.com/solvosci/slv-field-service",
+    "depends": ["fieldservice_person_salary", "fieldservice_skill"],
+    "data": ["views/fsm_order_salary_views.xml"],
+    "installable": True,
+}
```
Comment (ChristianSantamaria) [inline fieldservice_person_salary_skill/__manifest__.py]: Missing EOF line

**Review by ChristianSantamaria — APPROVED** [review]
Locally tested, LGTM
Some improvement:

**Comment by dalonsod** [comment]
Please squash commits and I'll merge it 😀

**Comment by dalonsod** [comment]
@ChristianSantamaria could you install it on BfnProd environment?

**Comment by ChristianSantamaria** [comment]
> @ChristianSantamaria could you install it on BfnProd environment?

Sure

**Comment by ChristianSantamaria** [comment]
Done 👍

**Pull request merged**